### ISSUE TITLE
Alarm settings Amazfit Bip

### ISSUE THREAD

**Issue by Buntbart** [Freeyourgadget/Gadgetbridge#1676]
I use Gadgetbridge v. 0.36.2 from F-Droid on a Samsung Galaxy S5 (klte) with LineageOS 16 (Android 9). 

I like to use the alarm function of my Amazfit Bip with firmware 1.1.5.16. 
I have two problems with that: 

1. In GB I can't edit the last three alarm times in the list. They always remain fixed at 6:30 a.m.. 

2. After a synchronization of the activity data sometimes an alarm at 2:00 a.m. is switched on by itself.

**Comment by ashimokawa** [comment]
I assume you missed the fact that the bip always resorts the alarm by time.

 And there is no two way sync yet. Meaning that when you go to alarm setting in GB, those which were turned on before, get switched on on the watch again even if you disabled them on the watch or a one-shot alarm was already done.

**Comment by Buntbart** [comment]
No, I didn't miss the sorting order on the watch and that is part of the problem. So I assume you are able to change the last three alarm times in GB from 6:30 a.m. to a different time?

**Comment by ashimokawa** [comment]
Yes I can change them all :(

**Comment by ashimokawa** [comment]
Umm could it be that you have the "reserve alarms for reminder " option set or so?

**Comment by Buntbart** [comment]
yes! I didn't recognize the connection between these two functions. Thank you, that solved  #1 of my issues. I will try to keep an eye on #2 to give more information.

**Comment by ashimokawa** [comment]
@Buntbart 
Probably you had an appointment at 14:00? ;)

I agree is is a bug, those should not be displayed when using that feature. And on bip we should do it differently anyway...
